docs/fsharp/style-guide/formatting.md

```diff
@@ -1040,210 +852,408 @@ with
     printfn "Something went wrong: %A" ex
 ```
 
-## Formatting function parameter application
+### Formatting named arguments
```

Comment:
Should this be an H2 instead of an H3?

Reply:
Went through and fixed all these, some H2 --> H3 and some H3 --> H2